commit 50367fb5e8ce
Author: Shane Caraveo <scaraveo@mozilla.com>
Date:   Tue Apr 17 10:06:13 2018 -0500

    Bug 1450565 use GET for initial authorization request, r=mossop a=jcristau
    
    MozReview-Commit-ID: 70PQSeoxvOF
    
    --HG--
    extra : source : e17c105c0be870ad5d07a1ff0b0a5da1646f35a7
---
 toolkit/components/extensions/ext-identity.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git toolkit/components/extensions/ext-identity.js toolkit/components/extensions/ext-identity.js
index db4ea3e80a06..ab7811774fb6 100644
--- toolkit/components/extensions/ext-identity.js
+++ toolkit/components/extensions/ext-identity.js
@@ -14,7 +14,7 @@ var {
 const checkRedirected = (url, redirectURI) => {
   return new Promise((resolve, reject) => {
     let xhr = new XMLHttpRequest();
-    xhr.open("HEAD", url);
+    xhr.open("GET", url);
     // We expect this if the user has not authenticated.
     xhr.onload = () => {
       reject(0);
